.gitignore

```diff
@@ -160,3 +160,4 @@ settings.json
 # model artifacts
 
 **/*.pkl
+/examples/datasets/cifar10_data/
```

Comment:
We have a `#datasets` section in the .gitignore file. Feel free to stick this in there or rename your folder to `cifar_data` so that its covered by the existing ignore file